florist/tests/api/test_launch.py

```diff
@@ -0,0 +1,50 @@
+import os
+import tempfile
+from functools import partial
+from pathlib import Path
+from typing import Callable, Dict
+
+import torch
+from fl4health.server.base_server import FlServer
+
+from florist.api.launch import launch
+from florist.tests.api.utils.fl4health_utils import MnistClient, get_server_fedavg
+from florist.tests.api.utils.models import MnistNet
+
+
+def fit_config(batch_size: int, local_epochs: int, current_server_round: int) -> Dict[str, int]:
+    return {
+        "batch_size": batch_size,
+        "current_server_round": current_server_round,
+        "local_epochs": local_epochs,
+    }
+
+
+def get_server(
+    fit_config: Callable[..., Dict[str, int]] = fit_config,
+    n_clients: int = 2,
+    batch_size: int = 8,
+    local_epochs: int = 1,
+) -> FlServer:
+    fit_config_fn = partial(fit_config, batch_size, local_epochs)
+    server = get_server_fedavg(model=MnistNet(), n_clients=n_clients, fit_config_fn=fit_config_fn)
+    return server
+
+
+def test_launch() -> None:
+    n_clients = 2
+    n_server_rounds = 2
+    server_address = "0.0.0.0:8080"
+
+    with tempfile.TemporaryDirectory() as temp_dir:
+        client_data_paths = [Path(f"{temp_dir}/{i}") for i in range(n_clients)]
+        for client_data_path in client_data_paths:
+            os.mkdir(client_data_path)
+        clients = [MnistClient(client_data_path, [], torch.device("cpu")) for client_data_path in client_data_paths]
+
+        launch(
+            get_server,
+            server_address,
+            n_server_rounds,
+            clients,
+        )
```

Comment:
Any way you can do a quick assertion on the output logs? Just search for a string saying "training has completed" or something like that.